app/modules/home/HomeScreen.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+const HomeScreen = () => {
+    
+  return (
+    <View>
```

Comment:
use <>  and  < / >, if there is no props

Reply:
resolved